pkg/controller/registry/reconciler/grpc.go

```diff
@@ -478,3 +495,29 @@ func (c *GrpcRegistryReconciler) podFailed(pod *corev1.Pod) (bool, error) {
 	}
 	return false, nil
 }
+
+// PodHashMatch will check the hash info in existing pod to ensure its
+// hash info matches the desired Service's hash.
+func PodHashMatch(existing, new *corev1.Pod) bool {
```

Comment:
Any reason this needs to be exported?

Reply:
It doesn't need to so I can lowercase it.